src/qibosoq/qick_programs.py

```diff
@@ -228,11 +242,11 @@ def acquire(
             debug (bool): if true prints the program actually executed
             average (bool): if true return averaged res, otherwise single shots
         """
-
+        if self.readouts_per_experiment is not None:
+            readouts_per_experiment = self.readouts_per_experiment
```

Comment:
Here you are overriding the `readouts_per_experiment` the user gives as an argument when calling the function. Is this really what you want?

Reply:
Yes, it ovverides the number of readout only in the case of multiplexed readout because 3 acquisition at the same time are considered by qick a single acquisition, but in qibolab (and in general considering pulse sequences) are multiple pulses.